src/util/Constants.js

```diff
@@ -74,14 +74,14 @@ exports.DefaultOptions = {
   /**
    * HTTP options
    * @typedef {Object} HTTPOptions
-   * @property {number} [version=7] API version to use
+   * @property {number} [version=8] API version to use
    * @property {string} [api='https://discord.com/api'] Base url of the API
    * @property {string} [cdn='https://cdn.discordapp.com'] Base url of the CDN
    * @property {string} [invite='https://discord.gg'] Base url of invites
    * @property {string} [template='https://discord.new'] Base url of templates
    */
   http: {
-    version: 7,
+    version: 8,
```

Comment:
In v8, the `Retry-After` header in rate limits is in seconds, not milliseconds, so these need to be updated:
https://github.com/discordjs/discord.js/blob/b8be7237b843285f2e9ecb12fdea45c38ad993a8/src/rest/RequestHandler.js#L118
https://github.com/discordjs/discord.js/blob/b8be7237b843285f2e9ecb12fdea45c38ad993a8/src/rest/RequestHandler.js#L140

Also, would this be a breaking change as gateway intents are required in v8?